File: python/GateTwo/level4.py
def addition(first_number, second_number):
    total = first_number + second_number
    return total


def check_even(number):
    if number % 2 == 0:
        return True
    else:
         return False


def square_of_number(number):
   square = number * number
   return square

def convert_to_fahrenit(number):
    fahrenit = number * 9/5 + 32
    return fahrenit 

def check_prime_number(number):
    for count in range (2, number):
        if(number % count == 0):
            return False
        else:
            return True
def find_largest(first_number, second_number, third_number):
    largest =first_number 
    if second_number > largest and second_number > third_number:
       largest =   second_number 
       return largest
    else:
         largest = third_number 
         return  largest
    
def simple_interest(principal,rate,time):
    simple_interest = (principal * rate * time) / 100
    return simple_interest

def multiplication_table(number):
    for count in range(1,13):
        return f"{number} x {count} = {number * count}"

def area_of_rectangle(length, width):
    area = length * width
    return area

def reverse_

File: java/GateTwo/Level4.java
public class Level4{
    public int addition (int firstNumber, int secondNumber){

    return firstNumber + secondNumber;

}
    public boolean isEven(int number){
    
    if (number % 2 == 0){
        return true;
    }
    else{
        return false;

    }

    }
    public int square(int number){
    int square = number * number;
        return square;


    }

    public double fahrenheit(double number){
      double fahrenheit = number * 9/5 + 32;
        return fahrenheit;

    }

//    public boolean isPrimeNumber(int number){
//        boolean isPrime = true;
//        for(int count = 2; count < number; count++){
//            if(number % count == 0)
//                return false;
//            
//
//       
//        else{
//            
//            return true;
//
//        }
//
//        return isPrime;
//    }
//
//}
//

     public int findLargest(int firstNumber, int secondNumber, int thirdNumber){
        int largest = firstNumber;
        if(secondNumber > largest && secondNumber > thirdNumber){
            largest = secondNumber;
            return largest;
}
        else{
            largest = thirdNumber;
            return largest;
        }
    }

    public double simpleInterest(double principal, double rate, double time){
      double simpleInterest = (principal * rate * time) / 100;
       return simpleInterest;

        }


    public int area(int length, int width){

        int area = length * width;
        return area;


    }


    public String reverse(String number){
    String reverse = ""
    for(int index = number.length -1; index >= 0; index--){
        reverse += number.charAt(index);
        

    }
        return reverse;

    }
}


